<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExperimentWidget</class>
 <widget class="QWidget" name="ExperimentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>524</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Settings Tab</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout" name="expSettingsGrid" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="9" column="3">
      <widget class="QCheckBox" name="transScaleRHSCheckBox">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="stitchLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Output Stitch Params</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="debugLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Debug</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="summationTypeLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>SummationType</string>
       </property>
      </widget>
     </item>
     <item row="14" column="3">
      <widget class="QPushButton" name="getExpDefaultsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>32</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Restore Defaults</string>
       </property>
       <property name="fixedWidth" stdset="0">
        <number>70</number>
       </property>
       <property name="fixedHeight" stdset="0">
        <number>100</number>
       </property>
       <layout class="QHBoxLayout" name="getExpDefaultsButtonLayout"/>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="polCorrCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="startOverlapEdit">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <widget class="MantidQt::MantidWidgets::WorkspaceSelector" name="floodWorkspaceWsSelector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="floodCorLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Flood Correction</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="analysisModeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>PointDetectorAnalysis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MultiDetectorAnalysis</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QDoubleSpinBox" name="endOverlapEdit">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="startOverlapLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Transmission Run Overlap Start</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="floodWorkspaceWsSelectorLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Flood Workspace</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="addPerAngleOptionsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Add another row to the per-angle options table</string>
       </property>
       <property name="whatsThis">
        <string>Adds another row to the per-angle options table so that additional angles can be added</string>
       </property>
       <property name="text">
        <string>Add row</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="polCorrLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Polarisation Corrections</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="reductionTypeComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DivergentBeam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NonFlatSample</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="includePartialBinsCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="analysisModeLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>AnalysisMode</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="summationTypeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>SumInLambda</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SumInQ</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="endOverlapLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Overlap End</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="transmissionStitchParamsLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Transmission Stitch Params</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="perAngleTableLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Per-angle defaults</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="floodCorComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Workspace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ParameterFile</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="reductionTypeLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>ReductionType</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="debugCheckBox">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1" rowspan="3" colspan="3">
      <widget class="QTableWidget" name="optionsTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Specify options based on the run angle. Leave the angle empty to specify defaults for all runs.</string>
       </property>
       <property name="whatsThis">
        <string>This table allows you to specify default values for runs that do not have values specified on the Runs tab. You can specify these on a per-angle basis, where the angle from the Runs table will be looked up in the this table and those options will be used where a match is found. You may also leave the angle blank in this table to specify the default options that should be used for all runs.</string>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>9</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Angle</string>
        </property>
        <property name="toolTip">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1st Trans Run(s)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2nd Trans Run(s)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Trans Spectra</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Q min</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Q max</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>dQ/Q</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Scale</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Run Spectra</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="transmissionScaleRHSLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Scale RHS Transmission</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="includePartialBinsLabel">
       <property name="minimumSize">
        <size>
         <width>117</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>IncludePartialBins</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="transStitchParamsEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::WorkspaceSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtWidgets/Common/WorkspaceSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
